Replicator/c4Replicator.hh

```diff
@@ -72,8 +78,39 @@ struct C4Replicator : public RefCounted,
 
     void setSuspended(bool suspended) {
         LOCK(_mutex);
-        if (!setStatusFlag(kC4Suspended, suspended))
+        if(_status.level == kC4Stopped) {
+            // Suspending a stopped replicator?  Get outta here...
+            Warn("Ignoring a suspend call on a stopped replicator...");
+            return;
+        }
+        
+        if(_status.level == kC4Stopping && !statusFlag(kC4Suspended)) {
+            // CBL-722: Stop was already called or Replicator is stopped,
+            // making suspending meaningless (stop() should override any
+            // suspending or unsuspending)
+            Warn("Ignoring a suspend call on a stopping replicator...");
```

Comment:
Do these really want to be Warnings?  There is no way to know the exact state of the replicator.  It seems like this will happen all the time...

Reply:
It won't.  It only happens if there are rapid calls to stop / start / suspend

Reply:
I see.  Still... is spamming the restart button a warning?

Reply:
Maybe not, it just bothers me....and this is actually not the restart button at this location but rapid backgrounding and foregrounding and backgrounding again.  I doubt the user will come out and say "Oh yeah I was licking the home button to see what happens" so if possible I'd like to log when we are in this unstable situation.

Reply:
ok.  sounds good to me.